packages/core/src/rules/common/info-license-strict.ts

```diff
@@ -0,0 +1,23 @@
+import { detectSpec } from '../../oas-types';
+import { Oas3Rule, Oas2Rule, Async2Rule, Async3Rule } from '../../visitors';
+import { validateDefinedAndNonEmpty, validateOneOfDefinedAndNonEmpty } from '../utils';
+
+export const InfoLicenseStrict: Oas2Rule | Oas3Rule | Async2Rule | Async3Rule = () => {
+  let specVersion: string | undefined;
+  return {
+    Root: {
+      enter(root: any) {
+        specVersion = detectSpec(root);
+      },
+      License: {
+        leave(license, ctx) {
+          if (specVersion === 'oas3_1') {
+            validateOneOfDefinedAndNonEmpty(['url', 'license'], license, ctx);
```

Comment:
I had to adjust like this to make the rule do what I expected.

```suggestion
            validateOneOfDefinedAndNonEmpty(['url', 'identifier'], license, ctx);
```

Reply:
Thanks! That was what I actually meant 😅

Reply:
Also fixed tests not failing for that.